nbodykit/plugins/datasource/ZeldovichSim.py

```diff
@@ -0,0 +1,96 @@
+from nbodykit.extensionpoints import DataSource
+from nbodykit import utils
+
+import numpy
+import logging
+
+logger = logging.getLogger('ZeldovichSim')
+         
+class ZeldovichSimDataSource(DataSource):
```

Comment:
I wonder if it is the time to add a FastPM python binding such that we can use some of the routines there, for 'sim' datasets.  e.g. the routine to fill the linear gaussian field with gadget random seeds could be useful here. Then we can also avoid reading the simulation from disk etc.